docs/framework-integration/angular.md

```diff
@@ -59,26 +62,24 @@ npx -p @angular/cli ng generate library component-library
 
 #### Creating a Stencil Component Library
 
-> If you already have a Stencil component library, skip this section.
+:::note
+If you already have a Stencil component library, skip this section.
+:::
 
-```bash
+```bash npm2yarn
```

Comment:
this `npm2yarn` bit here uses [a neat little plugin](https://docusaurus.io/docs/markdown-features/code-blocks#npm2yarn-remark-plugin) to add automatic translation of the contents of the code block from npm-ese to yarn-ian, like so:

<img width="1010" alt="Screen Shot 2022-08-09 at 3 02 46 PM" src="https://user-images.githubusercontent.com/6207644/183740415-77bfb5d5-8cba-457b-897c-614853672cf6.png">
<img width="979" alt="Screen Shot 2022-08-09 at 3 02 53 PM" src="https://user-images.githubusercontent.com/6207644/183740419-01afd6b1-ac82-4e34-9d81-18feceafdaaa.png">
<img width="982" alt="Screen Shot 2022-08-09 at 3 03 01 PM" src="https://user-images.githubusercontent.com/6207644/183740420-2de3c630-3bdc-494d-825d-d9244cf6952e.png">
<img width="979" alt="Screen Shot 2022-08-09 at 3 03 05 PM" src="https://user-images.githubusercontent.com/6207644/183740422-3a6509ad-bfd2-4ea8-96c5-a5a80667c6f4.png">